app/views/handlers/feedback.py

```diff
@@ -84,13 +83,16 @@ def get_page_title(self) -> str:
         return self.PAGE_TITLE
 
     def handle_post(self) -> None:
-        session_data = self._session_store.session_data
+        session_data: SessionData = self._session_store.session_data  # type: ignore
         session_data.feedback_count += 1
 
-        feedback_metadata = FeedbackMetadata(
-            session_data.case_id,
-            session_data.tx_id,
-        )
+        if isinstance(session_data.case_id, str) and isinstance(
```

Comment:
Should be removed. Will be fixed by fixing the types on SessionData/Questionnaire metadata.

Reply:
After changing metadata type to `Mapping[str, Any]` `tx_id and `case_id` are still optional `str` so I think this needs to stay.